### DISCUSSION TITLE
admin login

### DISCUSSION THREAD

**Discussion by peterjonhera** [opencart/opencart#10665] [Q&A] [answered]
hi, i can't login to my opencart admin, its showing me this

{"redirect":"https:\/\/example.com\/opencart\/admin\/index.php?route=common\/dashboard&user_token=5bt4f08da155efc4165673192df26e3c"}

**Comment by mhcwebdesign** [comment, marked as answer]
Please re-post on the OpenCart forum, and provide more details.

**Reply by peterjonhera** [reply]
thank you
